common/lib/dependabot/ecosystem.rb

```diff
@@ -0,0 +1,161 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+
+module Dependabot
+  class Ecosystem
+    extend T::Sig
+
+    class VersionManager
+      extend T::Sig
+      extend T::Helpers
+
+      abstract!
```

Comment:
does the versionmanager only applies to packageManagers or will it support languages too?